src/Sylius/Bundle/ShopBundle/Resources/views/Account/_main.html.twig

```diff
@@ -0,0 +1,9 @@
+<div class="ui fluid vertical menu">
+    <div class="header item">My account</div>
```

Comment:
Of course, it should be, but this is only a static template for now, so I don't bother about that in this PR.